### PR TITLE
Add screen reader messages

### PR DESCRIPTION
**Pull request by esamattis**
And some additional aria labeling.

### CODE REVIEW

**Comment by esamattis** [comment]
!release ui

**Comment by esamattis** [comment]
!release ui

File: packages/ui/src/translations.ts [hunk 1]
```diff
@@ -1,14 +1,27 @@
...
+	"aria-show-all": "Show all search results in the group {{group}}",
 	"all-results-shown": "All results shown",
 	"load-more": "Load more",
 	"go-back": "Back",
 	"aria-label-close-search": "Close search",
 	"aria-label-search-input": "Search input",
 	"no-results": "No results",
+	"sr-result-count": "Got {{count}} search results for terms {{terms}}",
+	"sr-search-instructions":
+		"Search shows search results automatically as you type. Search results can be browsed with tabulator. Search searches for results in different groups and displays group's search results from best to worst. Search opens to its own window which can be closed with ESC or with Close-button at the end of the search window.",
```
Comment (JoonasVaris) [inline packages/ui/src/translations.ts:13]:
```suggestion
		"Search shows search results automatically as you type. Search results can be browsed with tabulator. Search searches for results in different groups and displays group's search results from best to worst. Search opens to its own window which can be closed with ESC or with Close-button at the start of the search window.",
```

This is outdated, Close button is located before search bar

File: packages/ui/src/translations.ts [hunk 2]
```diff
@@ -23,16 +36,24 @@ export const TRANSLATIONS: Record<string, TranslationStrings> = {
...
 		"show-all": "Näytä kaikki",
+		"aria-show-all": "Näytä kaikki hakutulokset ryhmässä {{group}}",
 		"load-more": "Lataa lisää",
 		"aria-label-close-search": "Sulje haku",
 		"all-results-shown": "Kaikki tulokset näytetty",
 		"aria-label-search-input": "Hakukenttä",
+		"sr-result-count":
+			"Hakutuloksia {{count}} kappaletta hakusanalla {{terms}}",
 		"no-results": "Ei hakutuloksia",
+		"sr-search-instructions": "Kuvaus snip",
```
Comment (JoonasVaris) [inline packages/ui/src/translations.ts:47]: Missing instructions?

**Review by JoonasVaris — APPROVED** [review]

File: packages/ui/src/translations.ts (hunk 2 above)
Comment (esamattis) [reply packages/ui/src/translations.ts:47]: oops :D @JoonasVaris

**Comment by esamattis** [comment]
!release ui

File: packages/ui/src/translations.ts (hunk 1 above)
Comment (esamattis) [reply packages/ui/src/translations.ts]: Removed completely. Esc is enough.

**Pull request merged**